transport-native-epoll/src/main/java/io/netty/channel/epoll/AbstractEpollChannel.java

```diff
@@ -83,7 +79,7 @@ boolean isFlagSet(int flag) {
     }
 
     @Override
-    public final FileDescriptor fd() {
+    public final Socket fd() {
```

Comment:
What are your reservations? Are you concerned about exposing the socket functionality?